shared/js/ui/templates/grade-details.es6.js

```diff
@@ -1,98 +0,0 @@
-const bel = require('bel')
-const header = require('./shared/sliding-subview-header.es6.js')
-const siteRating = require('./shared/site-rating.es6.js')
-const siteRatingExplainer = require('./shared/site-rating-explainer.es6.js')
-const tosdrMessages = {'A': 'Good', 'B': 'Mixed', 'C': 'Poor', 'D': 'Poor'}
-
-module.exports = function () {
-  if (!this.model) {
-    return bel`<section class="sliding-subview
-    sliding-subview--has-fixed-header">
-      ${header('Grade Details')}
-    </section>`
-  } else {
-    return bel`<div class="site-info site-info--details card card--no-top-margin">
```

Comment:
The styles for `site-info--details` and `site-info--details__explainer` can also go. (They live in popup.scss.)